.github/actions/acceptance-tests/action.yml

```diff
@@ -0,0 +1,67 @@
+name: Acceptance tests
+description: Run the acceptance tests against a single version of k8s
+inputs:
+  k8s-version:
+    description: 'Kubernetes version to use for the kind cluster'
+    required: true
+  operator-tarball:
+    description: 'Tarball of the operator Docker container to use for the tests'
+    required: true
+  configurator-tarball:
+    description: 'Tarball of the configurator Docker container to use for the tests'
+    required: true
+  kind-cluster-name:
+    description: 'Name of the kind cluster to create and test against'
+    default: 'operator-k8s'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v4.1.7
+    - uses: actions/setup-node@v4.0.3
```

Comment:
Why are we installing node?

Reply:
because I grabbed this from an old action I had written to use as a base and forgot to strip it out 😬